locale/en.po

```diff
@@ -899,15 +899,797 @@ msgstr ""
+msgid "OACUTE"
+msgstr "Ó"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:106
+msgid "OCIRCUMFLEX"
+msgstr "Ô"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:107
+msgid "OTILDE"
+msgstr "Õ"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:108
+msgid "ODIAERESIS"
+msgstr "Ö"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:109
+msgid "MULTIPLY"
+msgstr "×"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:110
+msgid "OOBLIQUE"
+msgstr "Ø"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:111
+msgid "UGRAVE"
+msgstr "Ù"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:112
+msgid "UACUTE"
+msgstr "Ú"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:113
+msgid "UCIRCUMFLEX"
+msgstr "Û"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:114
+msgid "UDIAERESIS"
+msgstr "Ü"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:115
+msgid "YACUTE"
+msgstr "Ý"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:116
+msgid "THORN"
+msgstr "Þ"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:117
+msgid "SSHARP"
+msgstr "ß"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:118
+msgid "DIVISION"
+msgstr "÷"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:119
+msgid "YDIAERESIS"
+msgstr "ÿ"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:120
+msgid "ESCAPE"
+msgstr "Esc"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:121
+msgid "TAB"
+msgstr "Tab"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:122
+msgid "BACKSPACE"
+msgstr "Backspace"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:123
+msgid "ENTER"
+msgstr "Enter"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:124
+msgid "KPENTER"
+msgstr "Num Enter"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:125
+msgid "INSERT"
+msgstr "Insert"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:126
+#: ../src/microbe_stage/editor/OrganellePopupMenu.tscn:228
+msgid "DELETE"
+msgstr "Delete"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:127
+msgid "PAUSE"
+msgstr "Pause"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:128
+msgid "PRINT"
+msgstr "Print"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:129
+msgid "SYSREQ"
+msgstr "SysRq"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:130
+msgid "CLEAR"
+msgstr "Clear"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:131
+msgid "HOME"
+msgstr "Home"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:132
+msgid "END"
+msgstr "End"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:133
+msgid "LEFT"
+msgstr "Left"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:134
+msgid "UP"
+msgstr "Up"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:135
+msgid "RIGHT"
+msgstr "Right"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:136
+msgid "DOWN"
+msgstr "Down"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:137
+msgid "PAGEUP"
+msgstr "Page Up"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:138
+msgid "PAGEDOWN"
+msgstr "Page Down"
+
+#: ../src/engine/input/key_mapping/KeyNames.cs:139
+msgid "CAPSLOCK"
+msgstr "Capslock"
```

Comment:
`Caps Lock` as Wikipedia uses that and it will be more consistent with num and scroll lock.